idb-sys/src/transaction/mod.rs

```diff
@@ -71,7 +68,6 @@ impl Transaction {
         let closure = Closure::once(callback);
         self.inner
             .set_onabort(Some(closure.as_ref().unchecked_ref()));
-        self.abort_callback = Some(closure);
```

Comment:
Removing this will drop the closure from memory once this function completes. This may lead to undefined behaviour.